src/pretix/plugins/sendmail/templates/pretixplugins/sendmail/update_rule.html

```diff
@@ -0,0 +1,55 @@
+{% extends 'pretixcontrol/event/base.html' %}
+{% load i18n %}
+{% load static %}
+{% load bootstrap3 %}
+{% block title %}{% trans "Update Scheduled Email" %}{% endblock %}
```

Comment:
The headlines are also a little inconsistent, "Update Scheduled Email" vs "Delete Rule". We should be clear to the end user about the name of this feature :)